### PR TITLE
Add status not applicable

### LINKED ISSUE #71: Translation of "Not applicable"
For a reporting status

This will support an extra status that allows countries to declare an indicator not applicable. For example: some landlocked countries may not wish to report on some coastal related indicators.

### PR DESCRIPTION
**Pull request by dougmet**
Fixes #71.

All but French from Google Translate. French I double-checked against a dictionary for N/A. I'm not 100% on:

- [ ] am
- [ ] ar
- [x] de
- [x] en
- [x] es
- [x] fr
- [ ] ru
- [x] zh-Hans

### CODE REVIEW

**Comment by JustusCas** [comment]
"Nicht zutreffend" is a better translation for DE

**Comment by dougmet** [comment]
From a Spanish colleague: 
> In Spanish I would say “No aplicable” or “No se aplica”

I'll go with 1.

**Comment by brockfanning** [comment]
@holypeony What do you think of the Chinese translation here?

File: translations/zh-Hans/status.yml [hunk 1]
```diff
@@ -3,3 +3,4 @@ overall_reporting_status: 整体上报状况
 reported_online: 已在线上上报
 reporting_status: 上报状况
 statistics_in_progress: 统计在进行中
+not_applicable: 不适用
```
Comment (holypeony) [inline translations/zh-Hans/status.yml]: Approved!

**Comment by dougmet** [comment]
French confirmed

**Pull request merged**